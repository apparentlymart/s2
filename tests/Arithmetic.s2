
layerinfo "type" = "core";
layerinfo "majorversion" = "1";

function main() {
    var int first = 7;
    var int second = 5;
    println "$first + $second = " + ($first + $second);
    println "$first - $second = " + ($first - $second);
    println "$second - $first = " + ($second - $first);
    println "$first * $second = " + ($first * $second);
    println "$first / $second = " + ($first / $second);
    println "$first % $second = " + ($first % $second);
}
