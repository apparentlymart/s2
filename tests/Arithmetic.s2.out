7 + 5 = 12
7 - 5 = 2
5 - 7 = -2
7 * 5 = 35
7 / 5 = 1
7 % 5 = 2
